src/logtab.cpp

```diff
@@ -818,18 +818,25 @@ void LogTab::ExportToNewTab()
     std::sort(idxList.begin(), idxList.end(), [](QModelIndex a, QModelIndex b) {
         return a.row() < b.row();
     });
-    emit exportToTab(idxList);
+    emit exportToTab(idxList,"exported data");
 }
 
 void LogTab::ExportToNewTab(COL column)
 {
+    // Generate the list of selected values
     auto selectionList = ui->treeView->selectionModel()->selectedRows();
     QSet<QString> exportedValues;
     for (const auto& selected : selectionList) {
        QString value = selected.model()->index(selected.row(), column, selected.parent()).data().toString();
        exportedValues.insert(value);
     }
 
+    // Build the name for the new tab
+    QStringList sortedValues=QStringList::fromSet(exportedValues);
+    sortedValues.sort();
+    QString name = QString(GetColumnName(column)) + " " + sortedValues.join(",");
```

Comment:
How big can the Tabs in the UI grow?
If they the size is unbounded, then the string should be truncated, otherwise we could end with tabs that occupy most of the screen with long titles such as "Key begin-query,empty-query,msg,logical-query,end-query,qp-query-end".
Perhaps only use the first 3 values and then put an ellipsis? "Key begin-query,empty-query,..."

If the tabs have a limit on size, then ignore this feedback.

Reply:
I don't think the tab label has a limit, although it will auto truncate when all tab labels reach the window size.
If we are going to truncate the string ourselves, we could still keep the full strings in tab tool tips.

Reply:
Limited to 30 characters and added the full text as tooltip